packages/ui/src/components/accordion/accordion.vue

```diff
@@ -3,8 +3,6 @@
   import { computed, getCurrentInstance, onMounted, provide, ref, watch } from 'vue';
   import { ACCORDION_KEY } from './accordion.ts';
 
-  import './accordion.css';
```

Comment:
@leonid we faced an issue with global defined variables. This approach doesn't work when we have several versions of the same lib on the page(microfrontend case). Because they overwrite each other.

How do you think can we migrate to the approach in this MR - with fallback variables. That way this is almost the same - we use global style and allow user to overwrite